src/index.js

```diff
@@ -31,11 +31,21 @@ $("#friends").click(async () => {
     friends = await query.getFriends()
 
     $.each(friends, (i, friend) => {
-        //$("#friendList").prepend("<li>" + friend.fullname + "</li>")
+        $(".friends-list").prepend("<ul><button style='width:200px' id='Amigo"+i+"'>" + "Chat with " + friend.name + "</button></ul>")
```

Comment:
Would be better to have css styling in the style.css and add a class/id to the button instead

Reply:
> Would be better to have css styling in the style.css and add a class/id to the button instead
You are right. Since the css was being modifed with bootstrap i didn't know how to propertly add to it, but i will change this asap

Reply:
If I remember we had a self made .css too so we can add our own rules there without conflicting with bootstrap. @JavierUrones knows more about this than me